### ISSUE TITLE
[Windows Terminal - Command Palette]: 'Shortcut keys' in 'Command Palette' are not visible in high contrast themes for focused or hovered rows.

### ISSUE THREAD

**Issue by SaurabhNew** [microsoft/terminal#17914]
### Windows Terminal version

1.22.2362.0

### Windows build number

27695.1000

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 27695.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Pre-requisite:**
Settings>Accessibility>Contrast themes>Aquatic/Desert theme>Reopen the application.

**Repro steps:**
1. Open the 'windows terminal' application.
2. Follow the above-mentioned pre-requisite step.
3. Now open any application like 'Windows Terminal'.
4. Open command Palette by pressing 'Ctrl + Shift + P' keys.
5. 'Command Palette' will open.
6.  Now navigate to any row by up/down arrow keys or hover with mouse.
7. Observe the issue.

**User experience:**
Users relying on high contrast themes, especially those with visual impairments, may find it challenging to see the shortcut keys in the Command Palette. This issue impacts navigation efficiency, as users cannot easily identify or use the shortcuts for commands.

**MAS Reference Link:**
[MAS 4.3.1 – No Disruption of Accessibility Features.docx (sharepoint.com)](https://microsoft.sharepoint.com/:w:/s/accessibility/EegntkLK4KBBvzE1qsKpIoABG2UIxUY2y3uo1LomKoz_bg?e=wLbDEr)

**Attachment:**
[Shortcut keys is not visible in high contrast themes.zip](https://github.com/user-attachments/files/16973765/Shortcut.keys.is.not.visible.in.high.contrast.themes.zip)

### Expected Behavior

The shortcut keys for commands in the Command Palette should be clearly visible in high contrast themes when rows are focused or hovered. Proper contrast between the text and background should ensure that users can easily see the shortcut keys.

### Actual Behavior

The shortcut keys associated with the commands in the Command Palette are not visible or are poorly visible when focused or hovered in high contrast themes. The lack of proper contrast makes it difficult to see the shortcut keys, impairing navigation.
